spyder_profiler/__init__.py

```diff
@@ -3,4 +3,8 @@
 #==============================================================================
 # The following statement is required to register this 3rd party plugin:
 #==============================================================================
+from spyder.utils.qthelpers import qapplication
+# The line below is needed in order to prevent a segmentation fault.
+# See issue #4014
+app = qapplication()
```

Comment:
Maybe we can do this only if there is not already an instance of QApplication running?

```

if not QApplication.instance():
   app = QApplication()
```

I dont know if having an application there will cause any side effects